Library/brew.sh

```diff
@@ -220,7 +230,7 @@ update-preinstall() {
     # Hide shellcheck complaint:
     # shellcheck source=/dev/null
     source "$HOMEBREW_LIBRARY/Homebrew/cmd/update.sh"
-    homebrew-update --preinstall
+    brew update --preinstall
```

Comment:
The two comment lines and the `source` are obsolete now that this is no longer calling the `homebrew-update` function directly … fixed in 406fdbb391e844ea604c3eac1075c3f2562d07a9.